eclass/font.eclass

```diff
@@ -190,15 +180,15 @@ font_src_install() {
 		pushd "${FONT_S}" > /dev/null
 		insinto "${FONTDIR}"
 		for suffix in ${FONT_SUFFIX}; do
-			doins *.${suffix} || die "font installation failed" # TODO old EAPI cleanup
+			doins *.${suffix}
 		done
 		font_xfont_config
 		popd > /dev/null
 	fi
 
 	font_fontconfig
 
-	[[ -n ${DOCS} ]] && { dodoc ${DOCS} || die "docs installation failed" ; } # TODO old EAPI cleanup
+	[[ -n ${DOCS} ]] && dodoc ${DOCS}
```

Comment:
Why not `einstalldocs`?

Reply:
just after pushing it, I noticed that too

Reply:
in theory, this is a _slight_ behavioral change, because you could also be installing `READMEdos.txt`, which is unlikely in the generic case, but in 1/100 ebuilds, this will happen. Still better, also with respect to `HTML_DOCS` and being able to use arrays.